core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/UserApiServiceSpec.scala

```diff
@@ -355,7 +355,7 @@ class UserApiServiceSpec extends ApiServiceSpec {
           assertResult(StatusCodes.OK) {
             status
           }
-          assertResult(List(RawlsBillingProjectMembership(project1.projectName, ProjectRoles.Owner, CreationStatuses.Error, Option(s"[Failure enabling api ${billingProjectMonitor.projectTemplate.services(1)}: this failed]")))) {
+          assertResult(List(RawlsBillingProjectMembership(project1.projectName, ProjectRoles.Owner, CreationStatuses.Error, Option(s"[Failure enabling api ${billingProjectMonitor.projectTemplate.services(1)}: this failed]"), Option(services.gcsDAO.accessibleBillingAccountName)))) {
```

Comment:
this PR adds business logic of "if caller is not `ProjectRoles.Owner`,  do not return billing account name". Could you  add a test for that?